GOVERNANCE.md

```diff
@@ -65,109 +66,60 @@ including:
 * Conduct guidelines
 * Maintaining the list of additional Collaborators
 
-* [Current list of CTC members](./README.md#current-project-team-members)
+* [Current list of TSC members](./README.md#current-project-team-members)
```

Comment:
Micro-nit: The link should probably go to `./README.md#tsc-core-technical-committee` which will link to the same place as https://github.com/nodejs/node#ctc-core-technical-committee one the `ctc` is changed to `tsc` in the README doc. Totally not a blocking objection, just a tiny suggestion.